docs/UserGuide.md

```diff
@@ -289,17 +289,17 @@ For example:
 
 #### Redoing previously undone changes : `redo`
 
-If you decide that you did not need to undo a change, you can use this command to redo the change.
+If you decide that you do not need to undo a command, you can use `redo` to restore the command.
```

Comment:
I'm not sure if it would be good to somewhat distinguish between a change vs a command?

Reply:
I switched to command because we are undoing and redoing commands. How should I distinguish?

Reply:
If we want to suggest that we are undoing/redoing commands instead of changes, then we would have to rephrase the view change history as well since that would then imply that you are viewing a command history. But that itself could become confusing with the recall feature.